functions/src/scripts/converter.ts

```diff
@@ -0,0 +1,336 @@
+
+import * as admin from 'firebase-admin';
+import * as readInput from './readInput';
+
+
+const rawBulletin = readInput.readInputFile(); //read bulletin json file
+const bulletinJSON = JSON.parse(rawBulletin); //parse file into json 
+const version = readInput.getVersion(); //get version of bulletin
+//if no version in file name default to 1
+
+const firebaseConfig = {
```

Comment:
we should consider putting the config into its own file and exporting it, so we don't have to declare it everywhere.